controllers/multiclusterhub_controller.go

```diff
@@ -352,6 +350,12 @@ func (r *MultiClusterHubReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return ctrl.Result{Requeue: true}, err
 	}
 
+	// iam-policy-controller was removed in 2.11
+	result, err = r.ensureNoClusterManagementAddOn(multiClusterHub, "iam-policy-controller")
```

Comment:
You need to add the `iam-policy-controller` key and value to the `ClusterManagementAddOns` the global variable in api/v1/multiclusterhub_methods.go or else `ensureNoClusterManagementAddOn` will log `Detected unregistered ClusterManagementAddon component` and return early.